src/io/flutter/sdk/FlutterSdk.java

```diff
@@ -69,33 +61,9 @@ private FlutterSdk(@NotNull final String homePath) {
     this(homePath, FlutterSdkUtil.getSdkVersion(homePath));
   }
 
-  /**
-   * Returns the same as {@link #getGlobalFlutterSdk()} but much faster
-   */
   @Nullable
   public static FlutterSdk getFlutterSdk(@NotNull final Project project) {
```

Comment:
I believe we borrowed this pattern from the Dart plugin.  Is it unnecessary there too?